### DISCUSSION TITLE
Definition of decibel

### DISCUSSION THREAD

**Discussion by Ghostbird** [w3c/wcag#3552] [Q&A] [answered]
§ [Success Criterion 1.4.7 Low or No Background Audio](https://www.w3.org/TR/WCAG21/#low-or-no-background-audio) has a part that says:

>20 dB
>
>   The background sounds are at least 20 decibels lower than the foreground speech content, with the exception of occasional > sounds that last for only one or two seconds.
>
>    NOTE
>
>    Per the definition of "decibel," background sound that meets this requirement will be approximately four times quieter than the foreground speech content.

This confused me. As far as I know, the Bel is a base 10 logarithmic scale, and 20 decibel would therefore equal 2 bel or a factor of 10² or one-hundred. Which is much more in line with my understanding of a 20 decibel sound difference.

Hint: Think of the difference between a band with five musicians (all equally loud), where one plays a completely different song than the other four. Versus the same situation with a orchestra of 101 musicians where still one plays the wrong song. In the second case, the one musician that's off, can be considered low background noise. In the first case it's a terrible nuisance.

Is this an error in the NOTE section of the document?

**Comment by patrickhlauke** [comment]
Indeed, well spotted. 20dB indeed shakes out to something being 100 times louder/quieter, which now makes me wonder if it's the `20dB` or the note that are wrong as well... (i mean the `20dB` is normative, so even if that *was* not the correct/intended value, then it's now far too late and far too complicated to retrospectively change in errata, so only the note that explains it would be more flexible/open to change)

**Comment by JediLin** [comment]
I think that the signal-to-noise ratio here is based on Sound Pressure Level difference. SPL (dB) is calculated by 20×log(P/P0) where P0 is the reference pressure (by definition 20 micro pascals). So the 20 dB SNR means 20×log(P1/P0) - 20×log(P2/P0) = 20, therefore P1 should be 10 times P2. Not 4 times anyway.

Extra note: I would like to recommend the "[Back to Basics: Speech Audiometry](https://www.audiologyonline.com/articles/back-to-basics-speech-audiometry-6828)" article. Killion and colleagues (2004) suggest that a person with severe SNR loss would require more than 15 decibel signal-to-noise ratio to obtain 50% speech understanding. (QuickSIN test uses 25 dB SNR at max.)

**Reply by Ghostbird** [reply]
Ah, relative SPL difference would lead to a factor 10 loudness indeed.

**Comment by JAWS-test** [comment, marked as answer]
"four times" is from the linked article in the understanding: https://ds.gpii.net/content/about-decibels-db

> Perception of Loudness   (20dB = 4x). Interestingly, our perception of loudness is not the same as sound pressure level.   Although the actual formulae is somewhat complex,  as a rough rule of thumb, an increase of 10db SPL is perceived to be approximately twice as loud.  Thus a 20 Db gain would seem to be about 4 times as loud. And a 40 Db gain would seem to be about 16 times as loud.

**Comment by Ghostbird** [comment]
Thanks for the answers. I think the 20dB is correct, based on my personal "gut feeling", and @JediLin's citation. However, it's interesting that the note may be right when it comes to perception of loudness. As this is a psychoacoustic matter, it's probably based on research on a large amount of people. I've worked in audio engineering, so it's possible that my personal perception has been calibrated away from the "default" towards the absolute decibel factor used by audio equipment.

Maybe the note could be updated to read:

Per the definition of "decibel," background sound that meets this requirement will be _perceived_ approximately four times quieter than the foreground speech content.

**Reply by Ghostbird** [reply]
No wait, the part that states _per the definition of "decibel"_ still makes no sense. Because it's not the definition of decibel that leads to this conclusion.

It should be something like:

According to psychoacoustics, a background sound that meets this requirement will be perceived approximately four times quieter than the foreground speech content.

**Comment by patrickhlauke** [comment]
so the note is just badly handwaving things / not clarifying what it actually means. I'd suggest proposing a change to the definition's note in a PR

**Comment by GreggVan** [comment]
I suggest we use  DBA going forward to avoid confusion

**Reply by JediLin** [reply]
I think dB A is primary used to measure an environment. For example, to quantify noise exposure.
dB SPL is used to measure device output, dB HL to measure human hearing threshold, and dB SL to indicate perceived sound pressure level of particular ear. 

For SNR, just dB.

**Comment by Myndex** [comment]
- When measuring power: +20 dB is a factor of 100

- When measuring voltage or pressure: +20 dB is a factor of 10

- The human *perception* of +20 dB is approximately 4. 
  - I say approximately as with all things perception, there are a lot of "it depends".

### Background
As for measuring signal to noise, or speech in noise, that's another matter. And as far as using an SPL meter in A weighting -- I'm not going to open up that can of worms, and being as this is an AAA SC, it's not something that would be mandated (and I'd offer that it couldn't or practically all programming would fail).

<details><summary><b>CLICK HERE to reveal more minutiae on dB and sound mixing</b></summary>
..

When we are mixing the soundtrack to a film or TV show, such a thing would be a non-starter. Among other things, music tends to be much wider band than voice, which occupies a fairly narrow band -- music can be louder if we notch out room for the vocal band, for instance. And the total level affects the perception of contrasting sound levels. 

From what I gather, this SC is derived from a study dealing with hum and noise that was interfering because it was in the same frequency band as speech. That does not necessarily apply to music or background sounds like wind and birds. And in this SC we aren't instructed as to how slow/fast we need to be averaging the signal...

As far as A weighting, from what I've read regarding Speech in Noise testing, C weighting would be more appropriate. Though to be honest, the suggested method of determining levels is impractical—it means the entire program has to be played out of a reference speaker, voice only then BG only. But that's now how we do mixes. And mixes are dynamic.

If the reference was a VU meter that was slow to smooth out peaks, that might make more sense. But then the figure of 20 dB separation is not well supported. In a typical TV or film mix, the dialog, which is center channel (same level in left and right) tends to hang out around -5 dB VU. Music under, or other backgrounds if present, are going to sit around -12 to -18 dB, and this is fine, depending on if there are a lot of sounds competing in the same band (like other voices). 

But it also matters what the total level is, as far as how much separation 20 dB sounds like. Like other forms of perception contrast, the sum total of the sound as an absolute measure substantially affects the perception of a measured difference. If the vocal was at 40 dB SPL, and the music was at 20 dB SPL, the music would likely be inaudible or barely noticeable. But if the vocal was at 80 dB SPL, and the music was at 60 dB SPL, it would be a very different perception of separation.

I'm not going to get started on the Fletcher Munson Curve.

There I go again writing too much. Cheers.


...

</details>
